autoload/xolox/lua.vim

```diff
@@ -23,9 +23,16 @@ function! xolox#lua#includeexpr(fname) " {{{1
   return a:fname
 endfunction
 
+let s:lua_path_cache = ['', []]
 function! xolox#lua#getsearchpath(envvar, luavar) " {{{1
+  if exists('s:lua_path_cache') && s:lua_path_cache[0] == g:lua_path
```

Comment:
Unchecked use of g:lua_path here causes `E121: Undefined variable: g:lua_path` for me.

Is this `exists` check testing the wrong thing? s:lua_path_cache will always exist since it's defined above.

Reply:
> testing the wrong thing?

Seems so, amended.